src/main/java/net/test/interceptor/TimingInterceptorNano.java

```diff
@@ -34,8 +40,11 @@ public Object intercept(@Origin Method method, @SuperCall Callable<?> callable)
     try {
       return callable.call();
     } finally {
-      System.out.println(
-        commonActionArgs.addPrefix("TraceAgent (timing): `" + method + "` took " + (System.nanoTime() - start) + " nano"));
+      long end = System.nanoTime();
```

Comment:
```java
    long end = (this.logThresholdMs == 0) ? 0 : System.nanoTime();
```

Reply:
I think we don't need to add this change to the Timing interceptors as we are using the `end` variable anyways as part of the printout.

Original code was like this:
`commonActionArgs.addPrefix("TraceAgent (timing): `" + method + "` took " + (System.nanoTime() - start) + " nano"));`

And code now looks like this:
commonActionArgs.addPrefix("TraceAgent (timing): `" + method + "` took " + (end - start) + " nano"));

Let me know if I'm missing something.

Reply:
You are right! I missed that...